### PR TITLE
Refactor/stp remove position enum

### LINKED ISSUE #914: Refactor PositionEnum pair

### PR DESCRIPTION
**Pull request by jordiluong**
#914 
Changed position enum to separate variables

### CODE REVIEW

**Review by timovdk — COMMENTED** [review]
So we don't need Receive target anymore?

**Comment by jordiluong** [comment]
> So we don't need Receive target anymore?

No, I dont think we need that. It is only used for the Receive tactic and the only position it ever needs to know is where to move to, so we can use positionMoveTo. Agree?

**Comment by timovdk** [comment]
> > So we don't need Receive target anymore?
> 
> No, I dont think we need that. It is only used for the Receive tactic and the only position it ever needs to know is where to move to, so we can use positionMoveTo. Agree?

Yeah, sounds fine.

File: include/roboteam_ai/stp/StpInfo.h [hunk 1]
```diff
@@ -144,9 +144,19 @@ struct StpInfo {
     std::optional<world_new::view::BallView> ball;
 
     /**
-     * Tuple of the PositionType and the position of this target
+     * Position to move to
+     */
+    std::optional<Vector2> positionMoveTo;
```
Comment (RobotJente) [inline include/roboteam_ai/stp/StpInfo.h]: positionToMoveTo

Comment (jordiluong) [reply include/roboteam_ai/stp/StpInfo.h]: Oke baas

**Review by RobotJente — APPROVED** [review]

**Pull request merged**